differentiable_robot_model/robot_model.py

```diff
@@ -633,46 +688,52 @@ def _get_parent_object_of_param(self, link_name: str, parameter_name: str):
         body_idx = self._name_to_idx_map[link_name]
         if parameter_name in ["trans", "rot_angles", "joint_damping"]:
             parent_object = self._bodies[body_idx]
+            parent_object2 = self._bodies2[link_name]
```

Comment:
@exhaustin I don't fully understand what's happening here. It looks like we return the parent twice - why?